app/controllers/students/base_controller.rb

```diff
@@ -8,7 +8,7 @@ def must_be_student
   end
 
   def current_team
-    @current_team ||= current_user.teams.first
+    @current_team ||= current_user.teams.last
```

Comment:
We even [have](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/models/team.rb#L59) `in_current_season` ;-)